src/components/Header.tsx

```diff
@@ -1,16 +1,22 @@
 import liff from '@line/liff'
-import React from 'react'
+import React, { useContext, useMemo } from 'react'
 import styles from './Header.module.css'
 import Button from './Button'
+import { AppContext } from '../Context'
 
 export default function Header() {
+  const {filter, appId} = useContext(AppContext);
+  const appName = useMemo(() => {
+    return filter === 'LIFF' ? 'LIFF Playground' : 'LINE MINI App Playground'
```

Comment:
I noticed that the style of the Open in LINE button is broken.
Applying the following diff can prevent the button's style from breaking.

```diff
diff --git a/src/components/Header.module.css b/src/components/Header.module.css
index 87edea9..02e74ac 100644
--- a/src/components/Header.module.css
+++ b/src/components/Header.module.css
@@ -17,6 +17,14 @@
   align-items: center;
 }
 
+.left {
+  flex: 1;
+}
+
+.right {
+  flex-basis: 100px;
+}
+
 .left a {
   text-decoration: none;
   color: #000;
```
![localhost_3000__mini(iPhone SE)](https://github.com/user-attachments/assets/f7029423-46b0-4c84-853f-c1cfd3a8e52c)